common/logger_config.go

```diff
@@ -64,6 +64,20 @@ func DefaultLoggerConfig() LoggerConfig {
 	}
 }
 
+// DefaultTextLoggerConfig returns a LoggerConfig with the default settings for a text logger.
+// For use in tests or other scenarios where the logs are consumed by humans.
+func DefaultTextLoggerConfig() LoggerConfig {
```

Comment:
Could we add maybe `DefaultConsoleLoggerConfig". Only difference is NoColor = false. Then in cases where we run stuff locally we can use that, otherwise CI defaults to TextLogger?

Reply:
added